modules/sandboxed-containers-create-kataconfig-resource-web-console.adoc

```diff
@@ -66,8 +66,9 @@ spec:
   kataMonitorImage: registry.redhat.io/openshift-sandboxed-containers/osc-monitor-rhel8:1.2.0
   kataConfigPoolSelector:
     matchLabels:
-      <label_key>: '<label_value>'
+      <label_key>: '<label_value>' <1>
 ----
+<1> Labels in `kataConfigPoolSelector` only support single values; `nodeSelector` syntax is not supported.----
```

Comment:
After the sentence, the "----" shows up in the preview. Can you please remove that?